wire-ios/Wire-iOS/Sources/UserInterface/DeviceView/Views/Components/DeviceDetailsProteusView.swift

```diff
@@ -0,0 +1,65 @@
+//
+// Wire
+// Copyright (C) 2023 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import WireCommonComponents
+
+struct DeviceDetailsProteusView: View {
+    @ObservedObject var viewModel: DeviceInfoViewModel
+    @State var isVerfied: Bool
+
+    var body: some View {
+        VStack(alignment: .leading) {
+            CopyValueView(
+                title: L10n.Localizable.Device.Details.Section.Proteus.id,
+                value: viewModel.proteusID,
+                isCopyEnabled: false,
+                performCopy: nil
+            )
+            .padding(.all, ViewConstants.Padding.standard)
+            Divider()
+            Text(L10n.Localizable.Device.Details.Section.Proteus.activated)
+                .foregroundColor(SemanticColors.Label.textSectionHeader.swiftUIColor)
+                .font(UIFont.mediumSemiboldFont.swiftUIFont)
```

Comment:
You are still using in several places `UIFont.`

Reply:
done